ci/cloudbuild/builds/check-api.sh

```diff
@@ -123,7 +123,7 @@ function check_abi() {
       # characters in the string "internal", and it should again be followed
       # by some other number indicating the length of the symbol within the
       # "internal" namespace. See: https://en.wikipedia.org/wiki/Name_mangling
-      -skip-internal-symbols "(8internal|_internal)\d"
+      -skip-internal-symbols "(8internal|_internal|4grpc)\d"
```

Comment:
Should we add `absl` while we are at it?

Reply:
Part of me doesn't want to think too hard about what skipping `absl::optional<T>` means.

Reply:
> Have you had an `absl` false positive before?

Yes.

> Part of me doesn't want to think too hard about what skipping `absl::optional<T>` means.

Fair enough.

Reply:
I have an absl flake 

https://github.com/googleapis/google-cloud-cpp/pull/13596/checks?check_run_id=21537016008

Reply:
That seems like a true finding ...
```
absl::forward<std::__cxx11::basic_string<char>const&> ( remove_reference_t& t )
```
changed to ...
```
absl::forward<std::__cxx11::basic_string<char>&> ( remove_reference_t& t )
```
Perhaps that is not an "API change" however (just losing `const`), and regenerating the ABI dump is OK?

Reply:
Note that this is a template instantiation, so perhaps we lost and gained a symbol just by changing our own type?